projects/packages/my-jetpack/_inc/data/notices/use-notice.tsx

```diff
@@ -0,0 +1,16 @@
+import { useContext, useEffect } from '@wordpress/element';
+import { NoticeContext } from '../../context/noticeContext';
+
+const useNotice = ( { message, options, isError } ) => {
+	const { setCurrentNotice } = useContext( NoticeContext ) || {};
```

Comment:
Do we need `|| {}` here?

Reply:
Yes we do need that here because it is possible that NoticeContext is null, which would throw a destructuring error when trying to get a value of null

Reply:
Hmm, isn't it better then to define the context's initial state as:
`{ currentNotice: <initialNoticeAsInUseState>, setCurrentNotice: noop (or null) }`

Setting it `|| {}` doesn't look correct type-wise.

Just thinking aloud here, I guess it doesn't matter so much.